docs/source/delta-clustering.md

```diff
@@ -61,23 +61,53 @@ Use the `OPTIMIZE` command on your table, as in the following example:
 OPTIMIZE table_name;
 ```
 
+Liquid clustering is incremental, meaning that data is only rewritten as necessary to accommodate data that needs to be clustered. Data files with clustering columns that do not match data to be clustered are not rewritten.
```

Comment:
> Data files with clustering columns that do not match data to be clustered are not rewritten.

Is it better say `Already clustered data files with different clustering columns are not rewritten`?

Reply:
Also should we mention that we still cluster the zcubes that are not full ( i.e. not sealed zcubes)?

Reply:
> > Data files with clustering columns that do not match data to be clustered are not rewritten.
> 
> Is it better say `Already clustered data files with different clustering columns are not rewritten`?

Updated.

Reply:
> Also should we mention that we still cluster the zcubes that are not full ( i.e. not sealed zcubes)?

I think the `data is only rewritten as necessary` serves this purpose. I didn't want to mention too many implementation details.